docs/api-reference/deck.md

```diff
@@ -377,6 +377,27 @@ Notes:
 * The query methods offer more flexibility for developers to handle events compared to the built-in hover and click callbacks.
 
 
+## Member Variables
+
+#### metrics
+
+A map of various performance statistics for the last 60 frames of rendering. Statistics generally fall into two categories, count and time, and the relevancy of data provided for
+statistic will depend on it's category (e.g. "time"-based values are only relevant for statistics that include timing). Data provided for each statistic are:
+- `time`: total time spent on an operation (relevant for time statistics)
+- `count`: number or amount of some metric (relevant for count statistics)
+- `average`: average time spent per operation (relevant for time statistics)
+- `hz`: number of operations per second (relevant for time statistics)
+
+Some metrics gathered in deck.gl include:
+- 'Update Attributes': time spent updating attributes
+- 'Redraw Layers': number of layers drawn
+- 'setProps Time': time spent setting properties
+- 'Redraw Count': number of times the scene was rendered
+- 'pickObject Time': time spent picking an object
```

Comment:
If the stat is called `pickObject Time`, the value should be a single number that is the time. If the value is an object, the stat should just be called `pickObject`?

Reply:
The stat names are just from grepping where stats are currently used. I agree that it's confusing, but I can't break the current API for this update.